drivers/mpu9250/MPU9250.hpp

```diff
@@ -181,6 +181,9 @@ namespace DriverFramework
 #if defined(__DF_EDISON)
 // update frequency 250 Hz
 #define MPU9250_MEASURE_INTERVAL_US 4000
+#elif defined(__DF_RPI_SINGLE)
+// update frequency 400 Hz
+#define MPU9250_MEASURE_INTERVAL_US 1000
```

Comment:
Sorry, I forget to fix the comment.  This definition is needed to change frequency. I use 1000HZ for raspberry pi3 .If using raspberry 1 or pi zero,1000hz may too higher